cloud_tasks/snippets/list_queues.py

```diff
@@ -13,28 +13,31 @@
 # limitations under the License.
 
 # [START cloud_tasks_list_queues]
-def list_queues(project, location):
-    """List all task queues."""
+from google.cloud import tasks_v2
 
-    from google.cloud import tasks_v2
 
-    # Create a client.
-    client = tasks_v2.CloudTasksClient()
+def list_queues(project: str, location: str) -> list[str]:
```

Comment:
As mentioned in the other comment please use List to support Python 3.7 and 3.8.